tests/installation/partitioning_raid.pm

```diff
@@ -159,6 +159,11 @@ sub run {
     # create partitioning
     send_key(is_storage_ng() ? $cmd{expertpartitioner} : $cmd{createpartsetup});
 
+    if (sle_version_at_least '15') {
```

Comment:
This sub menu is not yet in leap, but it makes sense for storage_ng only, so we should put it in next if (is_storage_ng) block and additionally check if it's sle using is_sle method.